octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/field/DateTimeFieldEditor.java

```diff
@@ -96,7 +98,12 @@ public void mouseDown(MouseEvent e) {
         SelectionAdapter selectionListener = new SelectionAdapter() {
             @Override
             public void widgetSelected(SelectionEvent e) {
-                entityModelWrapper.setValue(new DateFieldModel(fieldName, getZonedDateTime()));
+                try {
+                    entityModelWrapper.setValue(new DateFieldModel(fieldName, getZonedDateTime()));
+                } catch (DateTimeException ignored) {
+                    // sometimes you can input an invalid month if you hover

```

Comment:
Shouldn't be there some kind of return or print or something here if it catches the exception?

Reply:
Well, if you input an invalid date, it would just ignore it's value.
It's a bug in the DateTime UI widget, that it actually allows you to write 13 for month, then it fires the selection listener. 
Basically, all this does is avoids printing a huge exception in the console.